kuma/search/fields.py

```diff
@@ -38,14 +30,14 @@ class SiteURLField(serializers.Field):
     def __init__(self, url_name, args=None, kwargs=None):
         self.url_name = url_name
         self.args = args or []
-        self.kwargs = kwargs or {}
+        self.kwargs = kwargs or []
         super(SiteURLField, self).__init__(source='*')
 
-    def to_native(self, value):
+    def to_representation(self, value):
         if not value:
             return None
-        args = [serializers.get_component(value, arg) for arg in self.args]
-        kwargs = dict((arg, serializers.get_component(value, arg))
+        args = [serializers.get_attribute(value, [arg]) for arg in self.args]
+        kwargs = dict((arg, serializers.get_attribute(value, [arg]))
                       for arg in self.kwargs)
```

Comment:
can we use a dict comprehension here instead? Every time I see `dict` I feel like there's a better way.